appstudio-utils/util-scripts/lib/fetch/cluster.sh

```diff
@@ -13,8 +13,17 @@ k8s-save-data() {
   oc get $namespace_opt $kind $name -o json > $file
 }
 
+_policy-config-from-configmap() {
+  oc get configmap ec-policy -o go-template='{{index .data "policy.json"}}' 2>/dev/null
+}
+
+_default-policy-config() {
+  echo '{"non_blocking_checks":["not_useful"]}'
+}
+
 # Placeholder. In future it will likely come from
 # an EnterpriseContractPolicy crd, see HACBS-244
 save-policy-config() {
-  echo '{"non_blocking_checks":["not_useful"]}' | jq > $( json-data-file config policy )
+  # Note: the namespace the task is running in needs to have the ec-policy ConfigMap
+  { _policy-config-from-configmap || _default-policy-config ; } | jq > "$( json-data-file config policy )"
```

Comment:
Can you elaborate on this syntax? I haven't seen before

Reply:
Not sure either - I might have written `(  .... || ..... ) |` is it functionally different to that?

Reply:
`{ ... }` is used to group two commands here so that output of either would be passed to `jq`, otherwise if it was `_policy-config-from-configmap || _default-policy-config | jq` only if `_default-policy-config` branch is executed it'll be piped to `yq`. The `;` is in there because of [SC1083](https://github.com/koalaman/shellcheck/wiki/SC1083)

Reply:
`( ... )` launches a new shell, this is to save on that, one `$PID` and a bit of memory less...

Reply:
& that's why folk usually do `( cd <somewhere>; <do something> )` so when the script continues `$CWD` is not changed...

Reply:
Cool, thanks.